docs/docs/references/resources/fragments/_api_resource_schema.mdx

```diff
@@ -0,0 +1,49 @@
+<table>
+  <thead>
+    <tr>
+      <td>
+        <strong>Property Name</strong>
+      </td>
+      <td>
+        <strong>Property Description</strong>
+      </td>
+      <td>
+        <strong>Required</strong>
+      </td>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>API Name</td>
+      <td>A user friendly name that can help you to identify the API resource.</td>
+      <td>true</td>
+    </tr>
+    <tr>
+      <td>API Identifier</td>
+      <td>
+        The uniq <a href="https://www.rfc-editor.org/rfc/rfc8707.html">API Resource Indicator</a>{' '}
+        value, with restrictions listed above.
+        <br />
+        Logto will use it to identify the requested API resource and grand the authorization tokens
+        accordingly.
+      </td>
+      <td>true</td>
+    </tr>
+    <tr>
+      <td>
+        Token expiration time <i>(in seconds)</i>
```

Comment:
`(in seconds)` should be in the description

Reply:
Not sure about this.  if keep it as (in seconds) format, it should come right after the name. Otherwise should turn it into a sentence to be able to fit in as a description.

Reply:
nit: just my personal opinion: the descriptive info may not be placed in the `name` column.